coursework/mobile/index.md

```diff
@@ -526,6 +526,23 @@ for both your client-side and server-side developers to reference.
   you have chosen to ignore certain practices (if any). You will be penalized if your design violates principles with no good reasons.
 </div>
 
+### Alternatives to REST
+
+Besides REST, there have been various systems introduced more recently that support alternative ways for client-server communication. 
+These systems have seen increased adoption by companies that benefit from the different approaches to exposing API. 
+
+Here are some popular alternatives to REST
+- [GraphQL](https://graphql.org/)
+- [gRPC](https://grpc.io/) (based off RPC architecture)
+
+<div class="box">
+  <strong class="milestone-counter">Milestone 5:</strong>
+  Explore one alternative to REST API (may or may not be from the list above). 
+  Give a comparison of the chosen alternative to REST (pros and cons, context of use, etc).
+  
+  Between REST and your chosen alternative, identify which might be more appropriate for the application you are building for this project. Explain your choice.
+</div>
+
 ## Phase 2: REST Server
```

Comment:
Will asking them to build a REST server make sense if we're asking them to evaluate REST? Would it make more sense to ask them to implement what they chose instead

Reply:
I actually put the new evaluate REST alternatives milestone after the `Design and document all your REST API` milestone, can swap this to make more sense

I'm also considering the case where they find an alternative better based off description, but choose to implement REST because of time constraints

Will expand the ``Design and document all your REST API`  milestone to allow for alternatives, where they have to show how they are utilizing it well

Reply:
Reorganized milestones and added an alternative milestone for REST documentation if they choose to use alternatives to REST 5265f09de62f142b76f2a76d46343e62fd0e74c7